package.json

```diff
@@ -223,6 +223,8 @@
 		"test": "npm run lint && npm run test-unit",
 		"test-e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
 		"test-e2e:debug": "wp-scripts --inspect-brk test-e2e --config packages/e2e-tests/jest.config.js --puppeteer-devtools",
+		"test-e2e:open-browser": "cross-env PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=80 wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
```

Comment:
With updates added to the documentation, do you think it's still valuable to have this script included here? It's not that much to type `npm run test-e2e:watch -- --puppeteer-interactive`.

Reply:
I agree. I don't think we need them any more. I didn't know that the options existed.

Reply:
I don't know why it wasn't documented :(

Reply:
It's how things work in software development. Computer wants everything but we sometimes miss one or two. That's why we work together to find those missing pieces.